src/js/lib/content/src/index.html

```diff
@@ -7,7 +7,7 @@
         <!-- FIXME Security Vulnerabilities: https://github.com/luna/ide/issues/226 -->
         <meta http-equiv="Content-Security-Policy" content="
             default-src 'self';
-            script-src  'self' 'unsafe-eval' data: https://*;
+            script-src  'self' 'unsafe-eval' 'unsafe-inline' data: https://*;
```

Comment:
This is bad. Can we do it another way around?

Reply:
Not so much, this inline expression is constructed in Engine because it needs to know ID of the code block to copy :/ hence it is a inline expression.

Reply:
We need to do it safe

Reply:
It'll be done safely :)

Reply:
Done, even better - moved the js-part to the IDE so that it is loaded only once, and doesnt need to be sent with the HTML file, so the doc html itself gets much much shorter!